src/operator/pad.cc

```diff
@@ -400,12 +400,65 @@ Operator *PadProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_shape,
 DMLC_REGISTER_PARAMETER(PadParam);
 
 MXNET_REGISTER_OP_PROPERTY(Pad, PadProp)
-.describe(R"code(Pad an array.
+.describe(R"code(Pads an input array with a constant or edge values of the array.
 
-Only supports 4-D and 5-D input array.
+.. note:: `Pad` is deprecated. Use `pad` instead.
+
+.. note:: Current Implementation only supports 4-D or 5-D input arrays with padding applied only on the lower
+   2 or 3 dimensions(expects higher 2 dimensions in `pad_width` to be zero).
```

Comment:
space before (
"expects higher 2 dimensions in `pad_width` to be zero" -> "higher 2 dimensions in `pad_width` must be zero"